### PR TITLE
New commands: Set-DbaDbSchema and Remove-DbaDbSchema

### LINKED ISSUE #7140: New commands: Set-DbaDbSchema and Remove-DbaDbSchema
### Summary of new feature
Following on from the addition of #7125 to complete the set of schema functions.
- Set-DbaDbSchema
- Remove-DbaDbSchema

### Proposed technical details (if applicable)
Add functionality to update schema properties (owner, anything else that can be set?) and to drop schemas.

### PR DESCRIPTION
**Pull request by lancasteradam**
- Set-DbaDbSchema allows the schema owner to be changed
- Remove-DbaDbSchema will remove an empty schema

## Type of Change

 - [ ] Bug fix (non-breaking change, fixes # )
 - [x] New feature (non-breaking change, adds functionality, fixes #7140 )
 - [ ] Breaking change (effects multiple commands or functionality, fixes # )
 - [x] Ran manual Pester test and has passed (`.\tests\manual.pester.ps1)
 - [ ] Adding code coverage to existing functionality
 - [x] Pester test is included
 - [ ] If new file reference added for test, has is been added to github.com/sqlcollaborative/appveyor-lab ?
 - [ ] Nunit test is included
 - [x] Documentation
 - [ ] Build system
 

### Purpose
 
Adding commands requested in #7140.

### Approach

These are basic utility commands to alter the schema owner or to drop an empty schema.

### Commands to test

See the integration tests.

### CODE REVIEW

File: functions/Remove-DbaDbSchema.ps1 [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The target database(s).
+
+    .PARAMETER SchemaName
```
Comment (jpomfret) [inline functions/Remove-DbaDbSchema.ps1:23]: This should just be Schema - looking at other functions we don't add Name

File: functions/Set-DbaDbSchema.ps1 [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        Login to the target instance using alternative credentials. Accepts PowerShell credentials (Get-Credential).
+
+        Windows Authentication, SQL Server Authentication, Active Directory - Password, and Active Directory - Integrated are all supported.
+
+        For MFA support, please use Connect-DbaInstance.
+
+    .PARAMETER Database
+        The target database(s).
+
+    .PARAMETER SchemaName
```
Comment (jpomfret) [inline functions/Set-DbaDbSchema.ps1:23]: Just Schema again please :)

**Review by jpomfret — COMMENTED** [review]
Looks good - just a couple of small changes

**Comment by lancasteradam** [comment]
Thanks @jpomfret, much appreciated.

**Review by jpomfret — APPROVED** [review]
Looks good - thank you!

**Pull request merged**